modules/service/src/main/resources/db/migration/V0051__gmos_south.sql

```diff
@@ -0,0 +1,123 @@
+
+--- GMOS SOUTH DETECTOR
+
+create table t_gmos_south_detector (
```

Comment:
Not really using any of this yet.  Will be required to add GMOS South Long Slit though.